examples/kubernetes/windows/apply.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+kubectl apply -f $DIR/csi-driver.yaml
+kubectl apply -f $DIR/secrets.yaml
+kubectl apply -f $DIR/pv.yaml
+kubectl apply -f $DIR/pvc.yaml
+sleep 5 # Allow enough time for pvc to be created.
```

Comment:
this shouldn't be necessary. pod should eventually succeed once pvc is created and bound